problemsets-mat223/modules/module14-exercises.tex

```diff
@@ -1,17 +1,42 @@
 \begin{exercises}
 	\begin{problist}
-		\prob  Example problem with parts
-		\begin{enumerate}
-			\item   The first part
-			\item   The second part
-		\end{enumerate}
-		\begin{solution}
-		Solutions to the parts
-			\begin{enumerate}
-				\item   $(2,4,8)$
-				\item   $(15,6,-21)$
-			\end{enumerate}
-		\end{solution}
-
+		\prob Let $\mathcal T:\R^2\to\R^2$ be defined by $\mathcal T\mat{x\\y}=\matc{3x-y\\x-\tfrac{1}{4}y}$.
+	Find the volume of $\mathcal T(C_2)$.
+	
+	    \prob Let $\mathcal S:\R^3\to\R^3$ be defined by $\mathcal S\mat{x\\y\\z}=\matc{2x+y+z\\x-\tfrac{1}{2}y\\z}$.
+	Find the volume of $\mathcal S(C_3)$.
+	    
+	    \prob Let $\mathcal T:\R^2\to\R^2$ be defined by $\mathcal T\mat{x\\y}=\matc{x+2y\\-x-y}$.
+	    \begin{enumerate}
+			\item   Draw a picture of the transformation $\mathcal T$ to determine its orientation.
+			\item  	Find $\det(\mathcal T)$.
+        \end{enumerate}
+        
+        \prob Let $\mathcal S:\R^3\to\R^3$ be defined by $\mathcal S\mat{x\\y\\z}=\matc{x-y+z\\z+x-\tfrac{1}{3}y\\z}$.
+        Find $\det(\mathcal S)$.
+	    
+        \prob Let $\mathcal P:\R^3\to\R^3$ be defined by $\mathcal P\mat{x\\y\\z}=\matc{0\\0\\x+y+z}$.
+        Find $\det(\mathcal P)$.
+        
+        \prob Let $\mathcal T:\R^2\to\R^2$ be defined by $\mathcal T\mat{x\\y}=\matc{x+2y\\-x-y}$. Let $\mathcal S:\R^2\to\R^2$ be defined by $\mathcal S\mat{x\\y}=\matc{-x-2y\\x+y}$.
+        Find $\det(\mathcal S\circ \mathcal T)$.
+        
+        \prob Use elementary matrices to find the determinant of $A=\mat{2&3\\4&5}$.
+        
+        \prob Let $A=\mat{1&2&0\\0&2&1\\1&2&3}$.
+        \begin{enumerate}
+			\item \label{Module14-q8}   Use elementary matrices to find $\det(A)$. 
+			\item  	Find $\det(A^{-1})$.
+			\item   Draw a picture of the parallelepiped given by the rows of $A$.
+			\item  	Draw a picture of the parallelepiped given by the columns of $A$.
```

Comment:
Are you sure? This seems really hard.

Reply:
Which question? The calculation part or drawing a picture part?

Reply:
Drawing the picture.

Reply:
True, I would add a R2 question for drawing as an alternative